dotcom-rendering/src/web/components/Picture.tsx

```diff
@@ -111,25 +97,71 @@ const getSizes = (
 	) {
 		// Showcase main media images (which includes numbered list articles) appear
 		// larger than in body showcase images so we use a different set of image sizes
-		return `(min-width: ${breakpoints.wide}px) 1020px, (min-width: ${breakpoints.leftCol}px) 940px, (min-width: ${breakpoints.tablet}px) 700px, (min-width: ${breakpoints.phablet}px) 660px, 100vw`;
+		if (breakpoint >= breakpoints.wide) return 1020;
+		if (breakpoint >= breakpoints.leftCol) return 940;
+		if (breakpoint >= breakpoints.tablet) return 700;
+		if (breakpoint >= breakpoints.phablet) return 700;
+		return breakpoint;
 	}
 
 	switch (role) {
 		case 'inline':
-			return `(min-width: ${breakpoints.phablet}px) 620px, 100vw`;
+			if (
+				breakpoint >= breakpoints.tablet &&
+				breakpoint < breakpoints.desktop
+			)
+				return 680;
```

Comment:
Looking at DCR, there is a breakpoint where the image width increases to aprx. 680px. This is brings DCR in parity with Frontend, which honoured this.